packages/dashboard/src/widget-resize-controller.js

```diff
@@ -0,0 +1,210 @@
+/**
+ * @license
+ * Copyright (c) 2019 - 2024 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
```

Comment:
The license needs to be updated to the commercial one.

Reply:
Done